integration/src/main/java/ai/djl/integration/tests/training/OptimizerTest.java

```diff
@@ -172,6 +173,36 @@ public void testAdam() {
         }
     }
 
+    @Test
+    public void testAdamW() {
+        TestRequirements.engine("PyTorch");
```

Comment:
Is it possible to remove this TestRequirements? In the integration tests, we have an additional criteria that an `UnsupportedOperationException` does not count as failing the test. The suite could be considered as ensuring that an engine, to what extent it implements the DJL API, does so correctly. Not implementing a part of it is then not considered a failure.

Ideally, can you throw the exception with other engines (ie MXNet) so the test will be fine even for all engines. It also helps to clarify to end users that it is a known gap that the AdamW engine will not work and possibly why (even if the answer is we didn't implement this yet).